MediaBrowser.Controller/Entities/BaseItem.cs

```diff
@@ -2909,7 +2950,7 @@ public bool UpdateRatingToItems(IList<BaseItem> children)
 
         public IEnumerable<BaseItem> GetThemeSongs()
         {
-            return ThemeVideoIds.Select(LibraryManager.GetItemById).Where(i => i.ExtraType.Equals(Model.Entities.ExtraType.ThemeSong)).OrderBy(i => i.SortName);
+            return ThemeSongIds.Select(LibraryManager.GetItemById).Where(i => i.ExtraType.Equals(Model.Entities.ExtraType.ThemeSong)).OrderBy(i => i.SortName);
```

Comment:
`Where(i => i.ExtraType.Equals(Model.Entities.ExtraType.ThemeSong))` this shouldn't be needed, no?

Reply:
No, I forgot to remove it